File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.py
class Solution:
    def maxProduct(self, nums: List[int]) -> int:
        maxnum = 0
        for i in range(len(nums)): # 0 1 2 3 
            for j in range(len(nums)): # 0 1 2 3
                if i != j and maxnum <= (nums[i]-1) * (nums[j]-1):
                    maxnum = (nums[i]-1) * (nums[j]-1)
        return maxnum

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        int maxNumber = 0;
        for (int i = 0; i < nums.length;i++){
            for (int j = 0;j < nums.length;j++){
                maxNumber = i!= j && (nums[i] -1) *(nums[j]-1) > maxNumber ? (nums[i]-1) * (nums[j]-1):maxNumber;
            }
        }
        return maxNumber;   
    }
}